brainscore/benchmarks/public_benchmarks.py

```diff
@@ -37,7 +37,22 @@ def _standard_benchmark(identifier, load_assembly, visual_degrees, stratificatio
     return NeuralBenchmark(identifier=f"{identifier}-pls", version=1,
                            assembly=assembly, similarity_metric=similarity_metric, visual_degrees=visual_degrees,
                            ceiling_func=lambda: ceiler(assembly_repetition),
-                           parent=None, paper_link='http://www.jneurosci.org/content/35/39/13402.short')
+                           parent=None, paper_link='http://www.jneurosci.org/content/35/39/13402.short',
+                           bibtex= """@article {Majaj13402,
```

Comment:
use different bibtex for different public benchmarks, the way it's written now would use the Majaj&Hong bibtex also for Freeman&Ziemba. I suggest treating those like e.g. the `visual_degrees` and specifying them in the respective benchmark files as constants and then importing.